templates/content.mustache

```diff
@@ -9,15 +9,9 @@
     		{{> title.mustache}}
     	{{/unless}}
 
-      <!--
-      <ul class="on-this-page fixed open">
-        <li><a href="#">Monkeys</a></li>
-        <li class="active"><a href="#">Fish</a></li>
-        <li><a href="#">Cats</a></li>
-        <li><a href="#">Dogs</a></li>
+      <ul class="on-this-page">
```

Comment:
We'll need a conditional around this because if there is no TOC, we won't want the UL to be on the page